Applications/WebPortal/WebApp/library/source/pages/account_directory_page/account_directory_page/cached_account_directory_model.tsx

```diff
@@ -20,6 +22,14 @@ export class CachedAccountDirectoryModel extends AccountDirectoryModel {
     return this._model.groups;
   }
 
+  public get createAccountModel(): CreateAccountModel {
+    return this._model.createAccountModel;
+  }
+
+  public get groupSuggestionModel(): GroupSuggestionModel {
+    return this._model.groupSuggestionModel;
+  }
+
   public async createGroup(name: string): Promise<Beam.DirectoryEntry> {
     const group = await this._model.createGroup(name);
     this._model.groups.add(group);
```

Comment:
This line likely does nothing, change this implementation to:

    return await this._model.createGroup(name);